util/tc.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+from __future__ import print_function
+from __future__ import absolute_import
+# -*- coding: utf-8 -*-
+
+import sys
+import os
+import stat
+import urllib
+
+TASKCLUSTER_SCHEME = os.getenv('TASKCLUSTER_SCHEME',
+                               'https://index.taskcluster.net/v1/task/project.deepspeech.deepspeech.native_client.master.%(arch_string)s/artifacts/public/native_client.tar.xz')
+
+def get_tc_url(arch_string=None):
+
+    assert arch_string is not None
+
+    return TASKCLUSTER_SCHEME % { 'arch_string': arch_string }
+
+def maybe_download_tc(target_dir=None, tc_url=None, progress=True):
+    def report_progress(count, block_size, total_size):
+        percent = int((count * block_size * 100) / total_size)
```

Comment:
Don't need the int() here with `__future__.division`, just use `//`.

Reply:
Done